pkg/cmd/build/view.go

```diff
@@ -96,9 +96,16 @@ func NewCmdBuildView(f *factory.Factory) *cobra.Command {
 					}
 				}
 				if len(buildAnnotations) > 0 {
-					summary += lipgloss.NewStyle().Bold(true).Padding(0, 1).Underline(true).Render("\n\nAnnotations")
 					for _, a := range buildAnnotations {
-						summary += annotation.AnnotationSummary(&a)
+						annotationCount := 0
+						if len(annotation.AnnotationSummary(&a)) < 230 {
```

Comment:
Is this special `230` something to do with the annotation box, etc?

Reply:
Yeah, the total `len` with an empty annotation is `227`, this allows for special chars that maybe use an additional byte or two

Reply:
Hmm, what if we used glamour to render this instead. It handles markdown, which is a superset of HTML. A quick test with a string with markdown and html anchor tags and it works nicely